LayoutTests/http/tests/push-api/subscribe-deny-permissions-on-prompt-expected.txt

```diff
@@ -1,9 +1,8 @@
 CONSOLE MESSAGE: Push notification prompting can only be done from a user gesture.
-CONSOLE MESSAGE: Push notification prompting can only be done from a user gesture.
 PASS: service worker permissionState was prompt
 PASS: document permissionState was prompt
 PASS: service worker subscribe was error: NotAllowedError
 PASS: document subscribe without user gesture was error: NotAllowedError
-PASS: document subscribe with user gesture was error: NotAllowedError
-PASS: document subscribe with consumed user gesture failed with user gesture error
+PASS: document subscribe with user gesture consumed the user gesture
```

Comment:
The difference in behavior here is worrisome. This is meant to be a refactoring that maintain pre-existing behavior. Mixing behavior changes and refactoring in the same patch is usually a recipe for having a bad time later.